projects/managed-hubs/incidents.md

```diff
@@ -75,34 +76,104 @@ Here is the process that we follow for incidents:
+This lets us use notes, status updates from pagerduty as well as messages from Slack easily in the incident report!
+
+1. **Ensure that the incident is resolved**.
+   If not, refer to the proper step in [](support:incident-response).
+   The incident needs to be resolved before a report can be generated.
+2. **Open the incident** in the PagerDuty web interface, and click the `New Postmortem Report` button on top.
+3. `Owner of the Review Process` should be set to the Incident Commander, or someone else they delegate to explicitly.
+4. `Impact Start Time` is our best guess for when the incident started (*not* when the report came in).
+   `Impact End Time` is when service was restored.
+   Best guesses will do!
+5. **Add Data Sources** that we will use to keep track of the actions that happened around the incident.
+   - Link to the slack channel we created for this incident as a "Data Source", filled in with an appropriate time to cover all the messages there.
+   - Add any other channels where there was conversation there about the incident (e.g., GitHub Issues or Pull Requests).
+   
+   Click `Save Data Sources` to populate the timeline below with messages from the slack channels.
+6. **Fill out the timeline**. The goal is to be concise but make it possible for someone reading it to answer "what happened, and when?".
+   See [](incidents:postmortem-timeline) for more information.
+7. **Fill out the "Analysis" section** to the extent possible.
+   In particular, the "Action Items" should be a list with items linked out to GitHub issues created for follow-up.
+   Perfection is the enemy of the good here. Save as you go.
+8. **Click "Save & View Report"** when you are done, and ask other members of the incident response team to review the incident report.
+   They might add missing context, additional action items / summary details, or redact information. The person listed as
+   the "Owner of the Review Process" is still responsible for making sure the rest of the process is completed.
+9.  After sufficient review, and if the Incident Commander is happy with its completeness, **mark the Status dropdown as "Reviewed"**.
+10. Download the PDF, and add it to the `2i2c/infrastrtucture` repository under the `incidents/` directory. This make sure our incidents are all *public*, so others can learn from them as well.
+
+% Is there a way to share incidents in a way that doesn't require adding a binary blob to our repository? I think this generates extra toil in a process that already has a lot of toil, and also adds some clunkiness to git-based workflows. For example, could we have a public Google Drive folder where we drag/drop incident reports?
+
+
+(incidents:postmortem-timeline)=
+### Writing an incident timeline
+
+Below are some tips and crucial information that is needed for a useful and thorough incident timeline.
+
+The timeline should include:
```

Comment:
I think the easiest thing we can do is link to pre-existing timelines that are well-written. Then people can just riff off of the structure of those

Reply:
@choldgraf i agree! once we have a few more incidents we can incorporate that here.